src/components/inputsOutputs/InputsOutputsPage.js

```diff
@@ -12,7 +12,7 @@ exports.submitInputOrOutput = function(el){
   return el.$('[ng-click="addField()"]').click();
 };
 exports.deleteInputOrOutput = function(el){
-  return el.$('[ng-click="checkRemoveField(key,field)"]').click();
+  return el.$('[delete-confirm="checkRemoveField(key,field)"]').click();
```

Comment:
since function is renamed we should change this as well.
checkRemoveField -> removeField